From 51931eb12c03de1db46993173b55d9f0add0a859 Mon Sep 17 00:00:00 2001
From: Rodrigo Moya <rodrigo@gnome-db.org>
Date: Tue, 20 Sep 2011 13:29:56 +0200
Subject: [PATCH] shell: Use gtk_widget_show instead of _show_all in
 gnome_control_center_show

Using gtk_widget_show_all will make all invisible widgets show up when a
second instance of the shell is run from the command line.

https://bugs.launchpad.net/gnome-control-center/+bug/854544
---
 shell/gnome-control-center.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/shell/gnome-control-center.c b/shell/gnome-control-center.c
index c47d5bf..af8387c 100644
--- a/shell/gnome-control-center.c
+++ b/shell/gnome-control-center.c
@@ -1125,5 +1125,5 @@ gnome_control_center_show (GnomeControlCenter *center,
 			   GtkApplication     *app)
 {
   gtk_window_set_application (GTK_WINDOW (center->priv->window), app);
-  gtk_widget_show_all (gtk_bin_get_child (GTK_BIN (center->priv->window)));
+  gtk_widget_show (gtk_bin_get_child (GTK_BIN (center->priv->window)));
 }
-- 
1.7.5.4

